test/core/RequestHandler.test.ts

```diff
@@ -43,27 +43,43 @@ describe('RequestHandler', () => {
       compareRequests(postedRequest, requests[0]);
     });
 
-    it('reverts when posting duplicate request', async () => {
-      const { handler, requests } = await loadFixture(deployRequestHandlerFixture);
+    it('allows posting duplicate request', async () => {
+      const { handler, requests } = await loadFixture(deployFixture);
+
+      const requestId = await handler.postRequest.staticCall(requests[0]);
+      await handler.postRequest(requests[0]);
+
+      // Second post of the same request should succeed and return the same ID
+      const duplicateRequestId = await handler.postRequest.staticCall(requests[0]);
+      expect(duplicateRequestId).to.equal(requestId);
+
+      // Verify the request data remains unchanged
+      const postedRequest = await handler.getRequest(requestId);
+      compareRequests(postedRequest, requests[0]);
+    });
+
+    it('allows posting duplicate request with fees', async () => {
+      const { handler, requests } = await loadFixture(deployFixture);
 
-      const requestId = await handler.deriveRequestId.staticCall(requests[0]);
       await handler.postRequest(requests[0]);
 
-      await expect(handler.postRequest(requests[0]))
-        .to.be.revertedWithCustomError(handler, 'RequestAlreadyExists')
-        .withArgs(requestId);
+      // When posting a duplicate request with fees, it should succeed
+      const fee = ethers.parseEther('1');
+      await expect(
+        handler.postRequest.staticCall(requests[0], fee, 0, 0, { value: fee }),
+      ).to.be.revertedWithCustomError(handler, 'FeeManagerRequired');
     });
```

Comment:
This test doesn't make sense? The description and comment say something different from the code.